provider.go

```diff
@@ -74,14 +265,76 @@ func getAPI(hostAddr string, maxRetries int) (client.API, error) {
 func Provider() terraform.ResourceProvider {
 	return &schema.Provider{
 		Schema: map[string]*schema.Schema{
-			"tunnel_address": &schema.Schema{
-				Type:     schema.TypeString,
-				Required: true,
+			"driver": &schema.Schema{
+			    Type:     schema.TypeString,
+			    Optional: true,
+			    Description: "Adapter used to execute fleetctl commands. Options include api and etcd.",
+				Default: "api",
+			},
+			"endpoint": &schema.Schema{
+			    Type:     schema.TypeString,
+			    Optional: true,
+			    Description: "Location of the fleet API if --driver=api. Alternatively, if --driver=etcd, location of the etcd API.",
+				Default: "unix:///var/run/fleet.sock",
+			},
+			"etcd_key_prefix": &schema.Schema{
+			    Type:     schema.TypeString,
+			    Optional: true,
+				Default: registry.DefaultKeyPrefix,
+			    Description: "Keyspace for fleet data in etcd (development use only!)",
+			},
+			"key_file": &schema.Schema{
+			    Type:     schema.TypeString,
+			    Optional: true,
+				Default: "/var/run/fleet.sock",
```

Comment:
Looks like the problem is here: when I get the error that `/var/run/fleet.sock` doesn't exist, it's because of this default. Looks like it should just be `""`.